### PR TITLE
add draggable_scrollable_controller

### PR DESCRIPTION
**Pull request by dickermoshe**
Added useDraggableScrollableController with tests

Didn't edit readme or changelog
Lmk if you want the tests changed

### CODE REVIEW

**Comment by davidmartos96** [comment]
I'd say this hook would benefit from the changes in https://github.com/rrousselGit/flutter_hooks/pull/415

Current implementation of `useDraggableScrollableController` is not disposing the `DraggableScrollableController`.

**Comment by dickermoshe** [comment]
@davidmartos96 Good point!
I copied this from Expandable tile controller. I forgot that that isn't disposed.
I'll fix later

**Comment by dickermoshe** [comment]
Fixed, if @rrousselGit plans on merging #415 I'll rebase and push

File: packages/flutter_hooks/lib/src/draggable_scrollable_controller.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
+part of 'hooks.dart';
+
+/// Creates a [DraggableScrollableController] that will be disposed automatically.
+///
+/// See also:
+/// - [DraggableScrollableController]
+DraggableScrollableController useDraggableScrollableController(
+    {List<Object?>? keys}) {
```
Comment (rrousselGit) [inline packages/flutter_hooks/lib/src/draggable_scrollable_controller.dart:8]: A little trailling comma would be great :)
```suggestion
    {List<Object?>? keys,}) {
```

File: packages/flutter_hooks/lib/src/draggable_scrollable_controller.dart [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return use(_DraggableScrollableControllerHook(keys: keys));
+}
+
+class _DraggableScrollableControllerHook
+    extends Hook<DraggableScrollableController> {
+  const _DraggableScrollableControllerHook({List<Object?>? keys})
+      : super(keys: keys);
```
Comment (rrousselGit) [inline packages/flutter_hooks/lib/src/draggable_scrollable_controller.dart:15]:
```suggestion
  const _DraggableScrollableControllerHook({super.keys});
```

**Review by rrousselGit — APPROVED** [review]
LGTM after a little format change :)

**Review by rrousselGit — CHANGES_REQUESTED** [review]
Actually, could you add it to the README? And the changelog too, under the form of:

```dart
## Unreleased patch

- Added <...>
```

**Comment by dickermoshe** [comment]
Done

File: packages/flutter_hooks/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+
```
Comment (rrousselGit) [inline packages/flutter_hooks/CHANGELOG.md:1]:
```suggestion
```

**Comment by rrousselGit** [comment]
LGTM thanks!

**Review by rrousselGit — APPROVED** [review]

**Pull request merged**